tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -901,6 +901,26 @@ def test_speech_to_text_leveraged(self):
         output = speech_recognizer(filename)
         self.assertEqual(output, {"text": "a man said to the universe sir i exist"})
 
+    @slow
+    @require_torch_gpu
+    def test_wav2vec2_conformer_float16(self):
```

Comment:
This is the error repro that was failing before @Vaibhavs10 - added a slow integration test to make sure this works after the fix